Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -186,9 +187,45 @@ extension XcodeModuleProtocol  {
         return buildSettings.map{ "\($0) = \($1);" }.joined(separator: " ")
     }
 
+    func pkgConfigArgs() -> (Xcc: [String], Xld: [String]) {
+        var xcc = [String]()
+        var xld = [String]()
+        for module in recursiveDependencies {
+            guard case let module as CModule = module, let pkgConfigName = module.pkgConfig else {
+                continue
+            }
+
+            do {
+                let pkgConfig = try PkgConfig(name: pkgConfigName)
+                xcc.append(contentsOf: pkgConfig.cFlags)
+                xld.append(contentsOf: pkgConfig.libs)
+            }
+            catch PkgConfigError.CouldNotFindConfigFile {
+                if let providers = module.providers,
+                    provider = SystemPackageProvider.providerForCurrentPlatform(providers: providers) {
+                    print("note: you may be able to install \(pkgConfigName) using your system-packager:\n")
+                    print(provider.installText)
+                }
```

Comment:
Maybe it would be best to replace the `SwiftModule` extension with this one, then? Then the callers o the old `pkgConfigArgs()` can just combine the two arrays.